appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 0.1.{build}
```

Comment:
Filed [appveyor/ci/#1040](https://github.com/appveyor/ci/issues/1040). We'll see what happens.